src/word-search/index.js

```diff
@@ -1,3 +1,52 @@
-function wordSearch(words, word) {}
+function wordSearch(words, word) {
+  let stack = [];
+  let item;
+  let index;
+  for (let i = 0; i < words.length; i += 1) {
+    for (let j = 0; j < words[i].length; j += 1) {
+      let letterIndex;
+      if (words[i][j] == word[0]) {
+        stack.push([i, j]);
+        letterIndex = 0;
+        let checked = []
+        checked.push([i, j].join(''))
+        while (stack.length !== 0) {
```

Comment:
the code looks too complex, put this while loop in a helper function, and then call that function, that way the wordSearch() function looks cleaner. Right now there are so many things going on in the wordSearch() function which makes it difficult to read